### PR TITLE
requesting further change of Readme (wireframes)

### LINKED ISSUE #3: create Readme

### PR DESCRIPTION
**Pull request by ruszkipista**

### CODE REVIEW

**Comment by mayaprog** [comment]
LGTM (Looks Good To Me)

File: README.md [hunk 1]
```diff
@@ -2,3 +2,6 @@
 Practise the usage of collaborative development
 
 ##  Collaborators
```
Comment (mayaprog) [inline README.md]: Let's rename this to Contributors

**Review by mayaprog — APPROVED** [review]
LGTM (Looks Good To Me)

File: README.md (hunk 1 above)
Comment (ruszkipista) [reply README.md]: Renamed as requested

**Pull request merged**